### PR TITLE
missed one THIS

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
Matches Doc PR https://github.com/mulesoft/docs-connectors/pull/45 in same branch.

### CODE REVIEW

**Comment by Mulesoft-Mysti** [comment]
Remember that "THIS" I fixed the other week? I missed one! Can you review/approve?

File: modules/ROOT/pages/common/common-note-mule4.adoc [hunk 1]
```diff
@@ -1,10 +1,8 @@
-= THIS
+= Mule 4 Behavior for Mule 3 Users
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Notes on Mule 4 behavior (for Mule 3 users):
-
 * The examples above use the File connector to read a file in the middle of the flow. The information posted through the `<http:listener>` component is written to each FTP or SFTP site multiple times. Because the component makes use of the xref:4.1@mule-runtime::streaming-about.adoc[repeatable streams feature], you do not need to worry about consuming the stream multiple times.
```
Comment (george-hoffer) [inline modules/ROOT/pages/common/common-note-mule4.adoc:6]: "The examples above use the File connector to read a file in the middle of the flow. " -- What examples?

Comment (george-hoffer) [reply modules/ROOT/pages/common/common-note-mule4.adoc:6]: Should also explain why repeatable streams means that users don't have to worry about consuming a stream multiple times

File: modules/ROOT/pages/common/common-note-mule4.adoc [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
 
 * The `<foreach>` component automatically goes through each line of the CSV file. In Mule 3, you first needed to transform the CSV file into a Java structure, but because Mule 4 is now Java agnostic, this works out-of-the-box.
```
Comment (george-hoffer) [inline modules/ROOT/pages/common/common-note-mule4.adoc:8]: What does it mean that the foreach component "goes thru each line of the CSV file" ?  What CSV file? Why would a user want to apply a foreach statement to every line in a CSV file? That sounds very impractical.

**Review by george-hoffer — COMMENTED** [review]
Please fix the issues listed below.

**Comment by Mulesoft-Mysti** [comment]
This is a doc that should be deleted, not published.

**Comment by george-hoffer** [comment]
The common folder was removed. This PR no longer has files to change. Closing.

**Pull request closed without merging**